src/main/groovy/oe/espresso/latte/AblUnit.groovy

```diff
@@ -390,10 +395,21 @@ class AblUnit extends BaseLatteSourceTask {
                 }
             }
 
-
             this.source.addToAntBuilder(delegate, 'fileset', AntType.FileSet)
-
+            // Generate code coverage stats for tests?
+            if (coverage) {
+                if (!profilerOutputDir) {
+                    profilerOutputDir = new File(project.buildDir, "profiler")
+                }
+                ant.Profiler([enabled     : true,
```

Comment:
I think this needs a profilerOutputDir.mkdirs() so the user doesn't have to do it manually.

Reply:
It my testing the folder is created, without needing to call `mkdirs()` . I can certainly add it in.